implot-sys/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![allow(non_camel_case_types)]
 #![allow(non_snake_case)]
 
-pub use imgui;
+// just for linking for tests
+use imgui_sys as _;
```

Comment:
I think I'd like to add a bit more documentation here - as far as I understand it, we don't need the higher-level imgui bindings in implot-sys, because implot-sys just links to the C bindings for implot itself, which obviously don't need the higher-level imgui bindings. 

If we only need this in the tests though, we can do `use  imgui_sys` in the test module itself I think.

Reply:
The tests I was referring to are the ones that are automatically generated by bindgen and then put into bindings.rs. I put a cfg attribute on the import to only use it for tests now.